src/en/release/1.8.0.md

```diff
@@ -0,0 +1,713 @@
+[[KYUUBI #5209](https://github.com/apache/kyuubi/pull/5209)] Reformat Scala source code crossing versions
+[[KYUUBI #5212](https://github.com/apache/kyuubi/pull/5212)] Fix configuration errors causing by helm charts of prometheus services
+[[KYUUBI #5214](https://github.com/apache/kyuubi/pull/5214)] Correct file path for scala tests in util-scala module
+[[KYUUBI #5234](https://github.com/apache/kyuubi/pull/5234)] Upgrade Jetty version from 9.4.51.v20230217 to 9.4.52.v20230823
+[[KYUUBI #5235](https://github.com/apache/kyuubi/pull/5235)] Daily deploy snapshot version on master and branch-1.7
+[[KYUUBI #5250](https://github.com/apache/kyuubi/pull/5250)] Bump Iceberg from 1.3.0 to 1.3.1
+[[KYUUBI #5252](https://github.com/apache/kyuubi/pull/5252)] Remove incubator link
+[[KYUUBI #5253](https://github.com/apache/kyuubi/pull/5253)] Upgrade testcontainers-scala from 0.40.12 to 0.41.0
+[[KYUUBI #5253](https://github.com/apache/kyuubi/pull/5253)] Supply testcontainers-scala-scalatest deps
+[[KYUUBI #5293](https://github.com/apache/kyuubi/pull/5293)] Upgrade snakeyaml from 1.33 to 2.2
+[[KYUUBI #5305](https://github.com/apache/kyuubi/pull/5305)] Fix get local log UT
+[[KYUUBI #5308](https://github.com/apache/kyuubi/pull/5308)] Ensure release using Java 8
+[[KYUUBI #5318](https://github.com/apache/kyuubi/pull/5318)] Pin maven-source-plugin 3.2.1
+[[KYUUBI #5358](https://github.com/apache/kyuubi/pull/5358)] Bump iceberg 1.4.0
+[[KYUUBI #5365](https://github.com/apache/kyuubi/pull/5365)] Don't use Log4j2's extended throwable conversion pattern in default logging configurations
+[[KYUUBI #5425](https://github.com/apache/kyuubi/pull/5425)] Add thread name and ID to log4j2-defaults.xml
+[[KYUUBI #5435](https://github.com/apache/kyuubi/pull/5435)] Improve Kyuubi On Kubernetes IT
+[[KYUUBI #5443](https://github.com/apache/kyuubi/pull/5443)] Add kyuubi-kubernetes-it module to reformat script
+[[KYUUBI #5444](https://github.com/apache/kyuubi/pull/5444)] Check style with profiles of Spark 3.4 and 3.5
+[[KYUUBI #5483](https://github.com/apache/kyuubi/pull/5483)] Fix deploy command for Scala 2.13 TPC-DS/H connectors
+[[KYUUBI #5483](https://github.com/apache/kyuubi/pull/5483)] Release Spark TPC-H/DS Connectors with Scala 2.13
+[[KYUUBI #5486](https://github.com/apache/kyuubi/pull/5486)] Bump Kafka client version from 3.4.0 to 3.5.1
+[[KYUUBI #5508](https://github.com/apache/kyuubi/pull/5508)] Upgrade download-maven-plugin from 1.6.8 to 1.7.1
+[[KYUUBI #5544](https://github.com/apache/kyuubi/pull/5544)] Release contains Spark 3.5 extension
+[[KYUUBI #5561](https://github.com/apache/kyuubi/pull/5561)] Binary distribution artifact ships database schema scripts
+[[KYUUBI #5561](https://github.com/apache/kyuubi/pull/5561)] Rename folder to db-scripts
+[[KYUUBI #5569](https://github.com/apache/kyuubi/pull/5569)] Fix pnpm-lock file version
+
+### Credits
+
+Last but not least, this release would not have been possible without the following contributors:
+
+* Alex Zou
```

Comment:
Let's sort contributors by their given names

Reply:
we already did that, except to unknown names https://github.com/apache/kyuubi/pull/5637